docs/content/docs/tutorials/sagemaker-example.md

```diff
@@ -0,0 +1,346 @@
+{{% /hint %}}
+
+Next, create a `training.yaml` file to specify the parameters for your SageMaker training job. This file specifies your SageMaker training job name, any relevant hyperparameters, and the location of your training and validation data. You can also use this document to specify which Amazon Elastic Container Registry (ECR) image to use for training. 
+
+```yaml
+printf '
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: TrainingJob
+metadata:
+  name: '$JOB_NAME'
+spec:
+  # Name that will appear in the SageMaker console
+  trainingJobName: '$JOB_NAME'
+  hyperParameters: 
+    max_depth: "5"
+    gamma: "4"
+    eta: "0.2"
+    min_child_weight: "6"
+    objective: "multi:softmax"
+    num_class: "10"
+    num_round: "10"
+  algorithmSpecification:
+    # The URL and tag of your ECR container
+    trainingImage: '$XGBOOST_IMAGE'
+    trainingInputMode: File
+  # A role with SageMaker and S3 access
+  roleARN: '$SAGEMAKER_EXECUTION_ROLE_ARN' 
+  outputDataConfig:
+    # The output path of your model
+    s3OutputPath: s3://'$SAGEMAKER_BUCKET' 
+  resourceConfig:
+    instanceCount: 1
+    instanceType: ml.m4.xlarge
+    volumeSizeInGB: 5
+  stoppingCondition:
+    maxRuntimeInSeconds: 86400
+  inputDataConfig:
+    - channelName: train
+      dataSource:
+        s3DataSource:
+          s3DataType: S3Prefix
+          # The input path of your train data 
+          s3URI: s3://'$SAGEMAKER_BUCKET'/sagemaker/xgboost/train
+          s3DataDistributionType: FullyReplicated
+      contentType: text/libsvm
+      compressionType: None
+    - channelName: validation
+      dataSource:
+        s3DataSource:
+          s3DataType: S3Prefix
+          # The input path of your validation data 
+          s3URI: s3://'$SAGEMAKER_BUCKET'/sagemaker/xgboost/validation
+          s3DataDistributionType: FullyReplicated
+      contentType: text/libsvm
+      compressionType: None
+' > ./training.yaml
+```
+
+Use your `training.yaml` file to create a SageMaker training job:
+
+```bash
+kubectl apply -f training.yaml
+```
+
+After applying your `training.yaml` file, you should see that your training job was successfully created:
+
+```bash
+trainingjob.sagemaker.services.k8s.aws/ack-xgboost-training-job-7420 created
+```
+
+You can watch the status of the training job with the following command:
+
+```bash
+kubectl get trainingjob.sagemaker --watch
+```
+
+It will a take a few minutes for `TRAININGJOBSTATUS` to be `Completed`.
+
+```bash
+NAME                            SECONDARYSTATUS   TRAININGJOBSTATUS
+ack-xgboost-training-job-7420   Starting          InProgress
+ack-xgboost-training-job-7420   Downloading       InProgress
+ack-xgboost-training-job-7420   Training          InProgress
+ack-xgboost-training-job-7420   Completed         Completed
+```
+
+If your training job failed, look to the `status.failureReason` after running the following command:
```

Comment:
lets talk about success in addition to failure case here. say to user something like,

to see the details about the training job, run `describe ....` if `status.trainingJobStatus` is `Completed`, find the model location under `status.modelArtifacts.s3ModelArtifacts`. If `status.trainingJobStatus` is `Failed` look for `status.failureReason`